models/Vote.js

```diff
@@ -5,7 +5,7 @@ var mongoose = require('mongoose');
 var voteSchema = mongoose.Schema({
 	issue_id: String,
 	user_id: String,
-	vote: Boolean
+	vote: { type: Boolean, required: '{VOTE must be defined}' }
```

Comment:
Might want to change this to 'VOTE must be required.' since the validation check here just checks to see if the value is provided, and undefined = falsy value.